### PR TITLE
'division' fix applied

### PR DESCRIPTION
**Pull request by vilnius-dev**
"from __future__ import division" added in each file that contains division

### CODE REVIEW

**Comment by ericvaandering** [comment]
This looks like the 2to3 fixer, not the old_div fixer from futurize was applied. 

But like I said, I think this division issue requires some actual thinking. Figuring out which version is most appropriate for each piece of code. I don't want to be stuck, 5 years from now, with a bunch of unnecessary dependencies on futurize's old_div

File: src/python/WMComponent/AgentStatusWatcher/ResourceControlUpdater.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 """
 Perform cleanup actions
 """
+from __future__ import division
 __all__ = []
```
Comment (amaltaro) [inline src/python/WMComponent/AgentStatusWatcher/ResourceControlUpdater.py]: All divisions in this module are integer, so L223-224, L234-236, L259-264 (//)

File: src/python/WMComponent/JobArchiver/JobArchiverPoller.py [hunk 2]
```diff
@@ -2,6 +2,7 @@
 """
 The actual jobArchiver algorithm
 """
+from __future__ import division
 __all__ = []
```
Comment (amaltaro) [inline src/python/WMComponent/JobArchiver/JobArchiverPoller.py]: Integer division in L248 (and int can be removed)

File: src/python/WMComponent/JobCreator/CreateWorkArea.py [hunk 3]
```diff
@@ -10,6 +10,7 @@
 Class(es) that create the work area for each jobGroup
 Used in JobCreator
 """
+from __future__ import division
 
 import os
```
Comment (amaltaro) [inline src/python/WMComponent/JobCreator/CreateWorkArea.py]: Integer division in L315

File: src/python/WMComponent/JobCreator/JobCreatorPoller.py [hunk 4]
```diff
@@ -2,6 +2,7 @@
 """
 The JobCreator Poller for the JSM
 """
+from __future__ import division
 __all__ = []
```
Comment (amaltaro) [inline src/python/WMComponent/JobCreator/JobCreatorPoller.py]: Integer division in L116

File: src/python/WMComponent/PhEDExInjector/PhEDExInjectorPoller.py [hunk 5]
```diff
@@ -40,6 +40,7 @@
 deleted from the site it was originally injected at.
 
 """
+from __future__ import division
```
Comment (amaltaro) [inline src/python/WMComponent/PhEDExInjector/PhEDExInjectorPoller.py]: Integer division in L90 (int and round can be removed)

File: src/python/WMComponent/TaskArchiver/CleanCouchPoller.py [hunk 6]
```diff
@@ -1,6 +1,7 @@
 """
 Perform cleanup actions
 """
+from __future__ import division
 import httplib
 import json
```
Comment (amaltaro) [inline src/python/WMComponent/TaskArchiver/CleanCouchPoller.py]: Integer division in L720 and 726

File: src/python/WMCore/Algorithms/MathAlgos.py [hunk 7]
```diff
@@ -6,6 +6,7 @@
 Simple mathematical tools and tricks that might prove to
 be useful.
 """
+from __future__ import division
```
Comment (amaltaro) [inline src/python/WMCore/Algorithms/MathAlgos.py]: Remove float from L61, L131, L188,

File: src/python/WMCore/BossAir/Plugins/MockPlugin.py [hunk 8]
```diff
@@ -4,6 +4,7 @@
 
 Base class for BossAir plugins
 """
+from __future__ import division
```
Comment (amaltaro) [inline src/python/WMCore/BossAir/Plugins/MockPlugin.py]: Integer division L225

File: src/python/WMCore/Credential/Proxy.py [hunk 9]
```diff
@@ -3,6 +3,7 @@
 _Proxy_
 Wrap gLite proxy commands.
 """
+from __future__ import division
```
Comment (amaltaro) [inline src/python/WMCore/Credential/Proxy.py]: Integer division in L213 (and remove int)

File: src/python/WMCore/DataStructs/MathStructs/ContinuousSummaryHistogram.py [hunk 10]
```diff
@@ -8,6 +8,7 @@
 
 @author: dballest
 """
+from __future__ import division
 import math
```
Comment (amaltaro) [inline src/python/WMCore/DataStructs/MathStructs/ContinuousSummaryHistogram.py]: Remove float from L164

File: src/python/WMCore/Database/CMSCouch.py [hunk 11]
```diff
@@ -9,6 +9,7 @@
 NOT A THREAD SAFE CLASS.
 """
 from __future__ import print_function
+from __future__ import division
```
Comment (amaltaro) [inline src/python/WMCore/Database/CMSCouch.py]: Integer division L1148

File: src/python/WMCore/JobSplitting/EventAwareLumiBased.py [hunk 12]
```diff
@@ -11,6 +11,7 @@
 
 @author: dballest
 """
+from __future__ import division
```
Comment (amaltaro) [inline src/python/WMCore/JobSplitting/EventAwareLumiBased.py]: Remove float L123, L172, L183, L274
Remove math.floor and float L183

**Review by amaltaro — CHANGES_REQUESTED** [review]
Roman, I made several comments along this PR. Please update it.
I didn't check the test/* files since they should all be fine, otherwise unit tests would fail.

**Comment by ticoann** [comment]
I think this division change has to have more testing and code review. We will go file by file approach with this.

**Pull request closed without merging**